src/caselawclient/models/documents.py

```diff
@@ -464,9 +478,9 @@ def enrich(self) -> None:
         """
         Request enrichment of the document
         """
-
+        now = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
I think this is going to be a subtle bug, because some datetimes coming back to the api client are going to be in naive and some will be datetime aware. We need to handle and test both cases.

Reply:
Hey dragon! aha, i've split that change out to another branch - will open a seperate PR for it. This one should now only have the changes related to the GRC subdivisions.